shared/domains/chelonia/chelonia.js

```diff
@@ -1693,9 +1708,17 @@ async function outEncryptedOrUnencryptedAction (
   if (opType === GIMessage.OP_ACTION_ENCRYPTED && !params.encryptionKeyId) {
     throw new Error('OP_ACTION_ENCRYPTED requires an encryption key ID be given')
   }
+  if (params.encryptionKey) {
```

Comment:
This new parameter is needed to designate a raw key to use for encryption in `OP_ATOMIC`. The current implementation of `GIMessage` / `encryptedData` / `signedData` / `chelonia` isn't advanced enough to automatically switch to the new key on outgoing messages. The workaround is designating the key as a raw key.